packages/vite/src/node/ssr/ssrModuleLoader.ts

```diff
@@ -257,7 +264,11 @@ async function nodeImport(
   try {
     const mod = await dynamicImport(url)
     return proxyESM(mod)
-  } catch {}
+  } catch (err) {
+    err.importee = id
+
+    throw err
+  }
```

Comment:
Is the refactor to the `mod` parameter needed here? Maybe we can revert to `importer` to simplify it.

Attaching `importee` is fine by me too, maybe we can put a comment here as well explaining why it's needed

Reply:
I forgot to undo the refactor to the `mod` :(

Comment added